eng/pipelines/templates/pipeline.yml

```diff
@@ -106,39 +108,39 @@ stages:
         swix:
           enabled: true
         optprof:
-          enabled: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
+          enabled: ${{ variables['isOfficialBuild'] }}
```

Comment:
you probably don't want to remove `succeeded()`, as there's no point installing the optprof plugin if a previous step already failed.

Reply:
Does this look right to you? I can't find documentation that this is supported within a variable. I also didn't see that the templates pass in a condition - but this is "compiling" at least:
https://github.com/NuGet/NuGet.Client/pull/5716/commits/81376bd6c90cf5206e1a542fa443617cb438f9fa#diff-ff6085ab5cf199ecbf9b59ba9a5854fdc078bdc1c7516d16c95f6bff5ecd070aR112